entrypoint.sh

```diff
@@ -6,24 +6,42 @@ response=$(curl \
         -H "Content-Type: application/json" \
         -s "https://api.cloudflare.com/client/v4/zones/$CLOUDFLARE_ZONE_ID/dns_records?name=$RECORD_NAME.$RECORD_DOMAIN")
 
-recordIds=$(jq -r  '.result[].id' <<< "${response}" ) 
+success=$(jq -r  '.success' <<< "${response}" )
```

Comment:
this doesn't match what I see returned by the API. probably revert.

take a look here https://api.cloudflare.com/#dns-records-for-a-zone-list-dns-records you can see 'success' will. just be true. but we want the list if ids in the result array.

Reply:
If my understanding is correct – we want the list of ids in the result array to check whether or not an IPFS record already exists on Cloudflare for `RECORD_NAME` because the API doesn't let us create new records just update them.

I think this check is still performed on line 19 and we still do get the record ids on line 26. This here is just to provide error information to the user in case the API call fails for any reason.